src/model/data_extractor.py

```
@@ -13,62 +13,68 @@ class DataExtractor:
             ValueError: データ形式が不正な場合
         """
-        xlsx_files: List[str] = [f for f in os.listdir(input_dir) if f.endswith('.xlsx')]
+        xlsx_files: List[str] = [
+            f for f in os.listdir(input_dir) if f.endswith(".xlsx")
+        ]
         if not xlsx_files:
-            raise FileNotFoundError(f"inputディレクトリにxlsxファイルが見つかりません: {input_dir}")
-        
+            raise FileNotFoundError(
+                f"inputディレクトリにxlsxファイルが見つかりません: {input_dir}"
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)